nbs/01_lhs_parsing.ipynb

```diff
@@ -34,20 +34,21 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.        such that each node or edge has the a dictionary of constraints - attr_name -&gt; (attr_type_str, attr_value).
</pre><p>"the a" no need for the "a"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>